cloudblue_connect.custom_adapter.rb

```diff
@@ -0,0 +1,694 @@
+#
+# This file is a part of the Ingram Micro’s CloudBlue Connect Workato Extension.
+
+#
+
+# @copyright (c) 2021. Ingram Micro. All Rights Reserved.
+
+#
+
+
+{
+  title: "CloudBlue Connect",
+
+  # API key authentication example. See more examples at https://docs.workato.com/developing-connectors/sdk/guides/authentication.html
+  connection: {
+    fields: [
+      {
+        name: "api_token",
+        label: "API Token",
+        hint: "You can find your API Token in <a href='https://portal.connect.cloudblue.com/integrations/tokens/' target='_blank'>Connect</a>",
+        optional: false
+      },
+    ],
+
+    authorization: {
+      type: "custom_auth",
+
+      # Calendly uses header auth (Authorization: <api key>)
```

Comment:
delete this comment, it has no relation to the connector, it's from Workato sample